plonky/Cargo.toml

```diff
@@ -46,7 +46,7 @@ ark-std = { version = "0.4.0", optional = true }
 env_logger = "0.10"
 
 [features]
-default = ["bellman_vk_codegen/multicore", "franklin-crypto/multicore", "ethabi", "recursive_aggregation_circuit/multicore"]
+default = ["bellman_vk_codegen/multicore", "franklin-crypto/multicore", "ethabi", "recursive_aggregation_circuit/multicore", "algebraic/default"]
```

Comment:
cargo build --release --features "opencl plonky/default recursion/default" # This command cannot include the default algebraic feature

Reply:
--features "A feature" --features "B features" ...